ptftplib/tftpserver.py

```diff
@@ -72,7 +77,8 @@ class TFTPServerConfigurationError(Exception):
     """The configuration of the pTFTPd is incorrect."""
 
 
-class TFTPServerHandler(SocketServer.DatagramRequestHandler):
+# noinspection PyPep8Naming
```

Comment:
PEP8 doesn't want your methods to be in Camelcase.  It wants `TFTPServerHandler.serveRRQ` to be `TFTPServerHandler.server_rrq`  I added that to make my linter stop whining because changing those method names would be API breakage.